app/views/fragments/giraffe/contributionsMotherLoadCampaign.scala.html

```diff
@@ -0,0 +1,97 @@
+@import com.gu.i18n.CountryGroup
+@import views.support.PageInfo
+@import views.support.CountryGroupImplicits._
+@import abtests.Allocation
+
+@(pageInfo: PageInfo = views.support.PageInfo(),
+    countryGroup: CountryGroup,
+    maxAmount: Option[Int] = None,
+    mainScript: Option[String] = None,
+    errorMessage:Option[String])(content: Html)
+
+@main(pageInfo, Some(countryGroup), maxAmount, mainScript) {
+    <div class='contributions--this-land-is-your-land'>
```

Comment:
This one still mentions the other campaign: `contributions--this-land-is-your-land`